episodes/assumptions.md

```diff
@@ -86,55 +86,60 @@ With that in mind, let's try to come up with some assumptions of our own!
 
 ### Challenge: identifying assumptions
 
-Take 5 minutes to list out as many assumptions as you can about the EIA 923 Puerto Rico data (`pr_gen_fuel_monthly.parquet`) in the [data directory](../data/).
+Take 5 minutes to list out as many assumptions as you can about the
+EIA 923 Puerto Rico data (`pr_gen_fuel_monthly.parquet`) in the [data directory](../data/).
 
 Please put them in the shared Google doc that your instructor prepared for you.
+This will serve as a foundation for future challenges in this lesson.
 
-The goal is to get past the obvious ones and start thinking of some un-obvious assumptions - no need to limit yourself to 'realistic' ones at this stage.
+The goal is to get past the obvious ones and start thinking of some un-obvious assumptions -
+no need to limit yourself to 'realistic' ones at this stage.
 
-:::::::: instructor
+Some prompts to get you started:
 
-Put these in a Google doc that you share with the class.
+* what problems have you run into in previous datasets?
+* if you were here for the data exploration episode,
+  what are some things you learned about the data then?
+* what are other people writing, does that trigger new thoughts about what to add?
 
+When we return, we'll talk about which things worked. 
+
+::::
+
+:::: instructor
```

Comment:
It's annoying that this displays below the challenge, but nesting instructor notes inside challenges doesn't seem to work so here we are.